Arkadia.xml

```diff
@@ -25480,6 +25480,7 @@ trigger_func_skrypty_ui_misc_ptakopodobny_inkantuje()</script>
 						<string>Tylko i wylacznie mieszkancy osady moga tam wejsc.</string>
 						<string>Probujesz przecisnac sie przez otwor na dach, ale ze wzgledu na brak miejsca, zmuszony jestes wrocic na dol.</string>
 						<string>Straznik ignoruje twoja prosbe.</string>
+						<string>Nie mozesz tam przejsc, gdy stoi tu obsada statku\.$</string>
```

Comment:
Potrzebny jest jeszcze typ, brakuje na dole w regexCodePropertyList.
Gdzies juz tak tez bylo dodane wczesniej, musze namierzyc i poprawic :)

Reply:
done

Reply:
To nie powinien być regexp, zmien na np exact match

Reply:
Start of the line lepiej

Reply:
W patchu, dodanym przed waszymi komentami, jest zmiana na start of line.